### ISSUE TITLE
Data warehouse for EA organizations/donations/people

### ISSUE THREAD

**Issue by OAGr** [QURIresearch/potential-projects#18]
Right now EA data is really not well organized. It would be great if we could have a strong data warehouse system for key data, particularly, longtermist data (this includes EA community work).

[EffectiveAltruismData.com](https://forum.effectivealtruism.org/posts/CQaNyJfsRFZhseiLZ/effectivealtruismdata-com-a-website-for-aggregating-and) is similar, but fairly small.

Also similar to issue: #12 

Some example data would include: 
The data tables listed in #12:
- Much of QURI's stuff, like our list of projects (future and previous)
- Our World In Data data tables
- Much of [Vipul Naik](https://donations.vipulnaik.com/)'s websites
- There are a bunch of one-off effective altruism tables out there.
- Michael Aird has a lot of databases/collections. [Here's one](https://forum.effectivealtruism.org/posts/twMs8xsgwnYvaowWX/database-of-orgs-relevant-to-longtermist-x-risk-work).

But also, a lot of structured data:
- Every relevant organization.
- For each, their employees, donations, blog posts, etc.
- Ideally, estimates of impact/quality for each (this would be done more on a forecasting platform, later on)

Similar projects include:
- [Donations portal](https://donations.vipulnaik.com/) of Vipul Naik
- [Crunchbase](https://www.crunchbase.com/)
- [IMDB Pro](https://pro.imdb.com)

**Comment by uvafan** [comment]
I agree that this would be valuable, but not sure if it's as high priority as some other work QURI could be doing. In particular, it's not clear that this is one of the most important bottlenecks to useful work being done at this stage. If you haven't already, it would be good to talk to Michael Aird for his take; that's probably the first thing I'd do.

**Comment by OAGr** [comment]
I think this is one of the most straightforward projects on the table. This means it's the easiest for someone else to do, but also means it could be the easiest to outsource and one of the most predictable (we have ideas about how hard it will be, how beneficial it will be). The VOI is correspondingly lower.
